eth/backend.go

```diff
@@ -251,6 +309,7 @@ func (s *Ethereum) StateDb() common.Database            { return s.stateDb }
 func (s *Ethereum) ExtraDb() common.Database            { return s.extraDb }
 func (s *Ethereum) IsListening() bool                    { return true } // Always listening
 func (s *Ethereum) PeerCount() int                       { return s.net.PeerCount() }
+func (s *Ethereum) PeerInfo() int                        { return s.net.PeerCount() }
```

Comment:
Absolutely not, thanks for spotting. Will fix in https://github.com/ethereum/go-ethereum/issues/512